packages/@ember/-internals/runtime/index.ts

```diff
@@ -1,10 +1,11 @@
 export { default as RegistryProxyMixin } from './lib/mixins/registry_proxy';
 export { default as ContainerProxyMixin } from './lib/mixins/container_proxy';
 export { default as Comparable } from './lib/mixins/comparable';
-export { NativeArray } from '@ember/array';
```

Comment:
Unrelated change?

Reply:
Split out to https://github.com/emberjs/ember.js/pull/20343.